src/coreclr-debug/activate.ts

```diff
@@ -28,6 +28,9 @@ export async function activate(thisExtension: vscode.Extension<CSharpExtensionEx
     } else if (!CoreClrDebugUtil.existsSync(_debugUtil.installCompleteFilePath())) {
         completeDebuggerInstall(platformInformation, eventStream);
     }
+
+    const factory = new DebugAdapterExecutableFactory(platformInformation, eventStream, thisExtension.packageJSON, thisExtension.extensionPath);
+    context.subscriptions.push(vscode.debug.registerDebugAdapterDescriptorFactory('coreclr', factory));
```

Comment:
We will also need a DebugAdapterDescriptor for DesktopCLR debugging.

`context.subscriptions.push(vscode.debug.registerDebugAdapterDescriptorFactory('clr', factory));`

Reply:
Doh, thanks!

Reply:
Added registration for 'clr'